ui/src/main/java/com/xmartlabs/bigbang/ui/common/recyclerview/BindingItemViewHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.xmartlabs.bigbang.ui.common.recyclerview;
+
+import android.support.annotation.NonNull;
+
+@SuppressWarnings("WeakerAccess")
```

Comment:
yes, because there is a warning which say that the interface could be package protected